bindings/rust/evmc-declare/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "evmc-declare"
```

Comment:
Should this be called declare or derive?

Reply:
This isn't a derive macro. It is an attribute macro. `derive`-mode macros auto-impl a trait, while this macro does not. It generates FFI code for calling the EVMC VM to which it is applied.

Reply:
Wasn't this a derive macro initially?